src/status_im/contexts/onboarding/syncing/progress/view.cljs

```diff
@@ -42,32 +39,40 @@
 
 (defn try-again-button
   [profile-color logged-in?]
-  [quo/bottom-actions
-   {:actions (if logged-in? :one-action :two-actions)
-    :blur? true
-    :button-one-label (i18n/label :t/recovery-phrase)
-    :button-one-props {:type                :primary
-                       :accessibility-label :try-seed-phrase-button
-                       :customization-color profile-color
-                       :container-style     {:flex 1}
-                       :size                40
-                       :on-press            navigate-to-enter-seed-phrase}
-    (if logged-in? :button-one-label :button-two-label)
-    (i18n/label :t/try-again)
-    (if logged-in? :button-one-props :button-two-props)
-    {:type                (if logged-in? :primary :grey)
-     :accessibility-label :try-again-later-button
-     :customization-color profile-color
-     :container-style     {:flex 1}
-     :size                40
-     :on-press            #(try-again logged-in?)}}])
```

Comment:
I refactored this code, but it's very interesting.

@ilmotta FYI, just in case you didn't know, this code has the following structure:
```clojure
{:foo              1
 :bar              2
 (if true :foo :x) 10
 (if true :bar :y) 20}
```

so weird, because this is invalid in Clojure, it throws (as expected), an exception due to duplicated keys, but in Clojurescript, it overrides the previously written keys:
```clojure
;; =>
 {:foo 10
  :bar 20}
```

as this may not work in a future release (idk) I refactored to be Clojure-valid

Reply:
Strange indeed @ulisesmac. If we see this type of code we can make an effort to refactor if possible because we have other ways, like using `cond->`.